test/integration/components/elasticsearch/main.py

```diff
@@ -0,0 +1,87 @@
+from fastapi import FastAPI
+import os
+import uvicorn
+import requests
+import json
+import time
+import sys
+
+app = FastAPI()
+HEADERS = {'Content-Type': 'application/json'}
+ELASTICSEARCH_HOST = "http://elasticsearchserver:9200"
+
+RETRY_TIMEOUT = 30  
+RETRY_INTERVAL = 2 
+REQUEST_TIMEOUT = 5 
+
+@app.get("/health")
+async def health():
+    HEALTH_URL = ELASTICSEARCH_HOST + "/_cluster/health"
+    start_time = time.time()
+    
+    while True:
```

Comment:
no need to retry here, this endpoint will be called in a loop by `waitForTestComponents`; I meant to do something simple here like (pseudocode):

```
def health:
    try:
        resp = get(/cluster/health)
        if resp.green:
            return "ok"
    except:
        pass
    return "ko"
```